Actions/Github-Helper.psm1

```diff
@@ -69,7 +69,21 @@ function InvokeWebRequest {
         if ($outfile) {
             $params += @{ "outfile" = $outfile }
         }
-        Invoke-WebRequest  @params -Uri $uri
+        try {
+            $result = Invoke-WebRequest  @params -Uri $uri
+        }
+        catch [System.Net.WebException] {
+            $response = $_.Exception.Response
+            $responseUri = $response.ResponseUri.AbsoluteUri
+            if ($response.StatusCode -eq 404 -and $responseUri -ne $uri) {
+                Write-Host "::Warning::Repository ($uri) was renamed or moved, please update your references with the new name. Trying $responseUri, as suggested by GitHub."
```

Comment:
I'm fine with the warning but did you consider if we should throw an error with this message instead? I just fear that the warning will be overlooked and the builds will keep targeting the old repo name

Reply:
I think the warning will allow people to rename a repository without breaking all other repos.
Making an error would cause all other apps to fail, which I think is unnecessary.